### PR TITLE
mutt-trim: Init at 2020-02-26

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Remains to be tested.

### CODE REVIEW

File: pkgs/applications/networking/mailreaders/mutt-trim/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    chmod +x $out/bin/mutt-trim
+  '';
+
+  postFixup = ''
+    patchShebangs $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "unclutter and normalize quoted text in an e-mail";
+    # no license yet
```
Comment (teto) [inline pkgs/applications/networking/mailreaders/mutt-trim/default.nix]: no homepage either

Comment (ryantm) [reply pkgs/applications/networking/mailreaders/mutt-trim/default.nix:30]: Seems like we should open an issue for the lack of license upstream and reference the issue. Homepage could be https://github.com/Konfekt/mutt-trim

**Review by doronbehar — COMMENTED** [review]
Otherwise LGTM.

File: pkgs/applications/networking/mailreaders/mutt-trim/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  src = fetchFromGitHub {
+    owner  = "Konfekt";
+    repo   = "mutt-trim";
+    rev    = "eeaacb886f78b00e704cb06faec7fd7d2dbb7caa";
+    sha256 = "0bl6msck0skys0vaaajax1ghbsidi6yvqzz8ns0vry41gmbyd176";
+  };
+
+  propagatedBuildInputs = [ perl ];
+
+  buildPhase = null;
```
Comment (doronbehar) [inline pkgs/applications/networking/mailreaders/mutt-trim/default.nix:16]: Not needed I think, stdenv will not do anything even without this.

**Pull request closed without merging**